src/kili/core/helpers.py

```diff
@@ -86,15 +86,20 @@ def is_url(path: object):
     return isinstance(path, str) and re.match(r"^(http://|https://)", path.lower())
 
 
-def format_json_dict(result: Dict, http_client: HttpClient) -> Dict:
-    """Formats the dict part of a json return by a GraphQL query into a python object.
+def __format_json_dict(result: Dict, http_client: HttpClient) -> Dict:
```

Comment:
why double underscore?

Reply:
I also thought we wanted to get rid of this ugly format_result/format_json/format_json_dict stuff?

Reply:
double underscore prevent the function to be imported in another file. It is to avoid this function to be reused somewhere else.

Reply:
I don't make a PR to remove this format result. it is still used in many queries and mutations. We will be able to simply delete it once we will have moved every function. We just have to make sure not to use it anymore

Reply:
> double underscore prevent the function to be imported in another file. It is to avoid this function to be reused somewhere else.

you can still import it: https://stackoverflow.com/questions/14168791/actual-implementation-of-private-variables-in-python-class

but since we plan to remove it anyway, we can keep it like that 👍

Reply:
> I don't make a PR to remove this format result. it is still used in many queries and mutations. We will be able to simply delete it once we will have moved every function. We just have to make sure not to use it anymore

yes what I meant in my comment is that we shouldn't use it for the "new" clean archi